std_msgs/String robot_name
std_msgs/String current_job
std_msgs/String ongoing_action
std_msgs/String previous_action
std_msgs/String robot_position
std_msgs/String cart_present
std_msgs/Bool charging
std_msgs/Float64 battery